packages/airnode-examples/integrations/weather-multi-value/request-utils.ts

```diff
@@ -0,0 +1,33 @@
+import { encode } from '@api3/airnode-abi';
+import { cliPrint, getDeployedContract, readIntegrationInfo } from '../../src';
+
+// OpenWeather historical API allows free access only to the last 5 days
+// Use date for yesterday, excluding milliseconds
+const dt = Math.floor((Date.now() - 86400000) / 1000);
```

Comment:
It's a good practice to let the computer compute the result (this better communicates the expression value)

```suggestion
const dt = Math.floor((Date.now() - 24 * 60 * 60 * 1000) / 1000);
```